pkg/metrics/metrics.go

```diff
@@ -78,6 +77,25 @@ func New(log logr.Logger, c clock.Clock) *Metrics {
 			},
 		)
 
+		// The clockTimeSeconds metric was first added, however this was
+		// erroneously made a "counter" metric type. Time can in fact go backwards,
+		// see:
+		// - https://github.com/jetstack/cert-manager/issues/4560
+		// - https://www.robustperception.io/are-increasing-timestamps-counters-or-gauges
+		// In order to not break users relying on the `clock_time_seconds` metric,
+		// a new `clock_time_seconds_gauge` metric of type gauge is added which
+		// implements the same thing.
```

Comment:
Shall we add a "DEPRECATED: use clockTimeSecondsGauge instead" prefix to the help of `clockTimeSeconds` ?

Reply:
I agree, thanks :+1: